src/applications/mhv/medical-records/containers/Allergies.jsx

```diff
@@ -109,6 +109,11 @@ const Allergies = props => {
             value: item.notes,
             inline: !item.notes,
           },
+          {
```

Comment:
The "Observed..." field should come before Provider Notes. (for List and Details) See the sketch link in the ticket. https://www.sketch.com/s/6ef3de45-2534-4584-a8a6-b9dce212ad90/a/Oen4Oww